test_scripts/API/Navigation/UnsubscribeWayPoints/010_UnsubscribeWayPoints_Transfering_of_HMI_error_resultCode_to_mobile.lua

```diff
@@ -101,19 +101,25 @@ runner.Step("Activate App", common.activateApp)
 
 runner.Title("Test")
 runner.Step("Result Codes", common.printResultCodes, { resultCodes })
+
 runner.Title("Successful codes")
-for _, code in pairs(resultCodes.success) do
-  runner.Step("UnsubscribeWayPoints with " .. code .. " resultCode", unsubscribeWayPointsSuccess, { code })
+for _, item in pairs(resultCodes.success) do
+  runner.Step("UnsubscribeWayPoints with " .. item.hmi .. " resultCode", unsubscribeWayPointsSuccess, { item })
 end
 
 runner.Title("Erroneous codes")
-for _, code in pairs(resultCodes.failure) do
-  runner.Step("UnsubscribeWayPoints with " .. code .. " resultCode", unsubscribeWayPointsUnsuccess, { code, false })
+for _, item in pairs(resultCodes.failure) do
+  runner.Step("UnsubscribeWayPoints with " .. item.hmi .. " resultCode", unsubscribeWayPointsUnsuccess, { item, false })
 end
 
 runner.Title("Unexpected codes")
-for _, code in pairs(resultCodes.unexpected) do
-  runner.Step("UnsubscribeWayPoints with " .. code .. " resultCode", unsubscribeWayPointsUnsuccess, { code, true })
+for _, item in pairs(resultCodes.unexpected) do
+  runner.Step("UnsubscribeWayPoints with " .. item.hmi .. " resultCode", unsubscribeWayPointsUnsuccess, { item, true })
+end
+
+runner.Title("Unmapped codes")
```

Comment:
@dboltovskyi , please remove this check to separate script

Reply:
@Itileda Please check 16104b6